content/applications/inventory_and_mrp/inventory/management/products/usage.rst

```diff
@@ -5,68 +5,69 @@ When Should you Use Packages, Units of Measure or Special Packaging?
 
-.. image:: usage/usage_04.png
-    :align: center
+.. image:: usage/uom-conversion-by-category.png
+   :align: center
+   :alt: Units of Measure conversions happen automatically as long as units are in the same category.
 
 Packages
 ========
 
-The package is the physical container in which you put one or several
-products from a picking. For example, when you deliver a product, you
-can decide to separate the quantity into two different packages. It then
-allows you to have a report with the quantity of products for each
-package.
+The package is the physical container in which you put one or several products from a picking. For
+example, when you deliver a product, you can decide to separate the quantity into two different
+packages. It then allows you to have a report with the quantity of products for each package.
 
-To separate a delivery into different packages you will have to set the 
-done quantity to the desired package quantity then click on "PUT IN 
-PACK", do this for each package.
+To separate a delivery into different packages you will have to set the done quantity to the
+desired package quantity then click on "PUT IN PACK", do this for each package.
 
-.. image:: usage/usage_05.png
-    :align: center
+.. image:: usage/separate-delivery-into-different-packages.png
+   :align: center
+   :alt: Separate delivery into different packages
 
-.. image:: usage/usage_06.png
-    :align: center
+.. image:: usage/delivery-package-details.png
+   :align: center
+   :alt: Separate delivery package details
 
-Packaging
-=========
+Packagings
+==========
 
-The packaging is the physical container that protects your product. If
-you are selling computers, the packaging contains the computer with the
-notice and the power plug.
+The packaging is a standard container containing several units of your  product. If you are selling
+cans of soda, the packagings would be a 6-Pack, a 15-Pack or even a pallet.
 
-In Odoo, packagings are used for indicative purposes on sale orders.
-They can be specified on the product form, in the inventory tab.
+In Odoo, packagings are used for indicative purposes on sales/purchase orders and transfers. The
+main difference between packagings and units of measure is that packagings are defined at product
+level while UoMs are generic.
 
-.. image:: usage/usage_07.png
-    :align: center
+.. image:: usage/product-packaging-examples.png
+   :align: center
+   :alt: Different product packaging examples
 
-.. image:: usage/usage_08.png
-    :align: center
+.. image:: usage/package-field-on-po.png
+   :align: center
+   :alt: Package field on purchase order
 
 .. note::
-        Another useful use of the packaging is for product reception. By
-        scanning the barcode of the packaging, Odoo adds the number of units
-        contained in the packing on the picking.
+   Another useful use of the packaging is for product reception. By scanning the barcode of the
```

Comment:
- Replace `Another useful...product reception.` with `Packaging is also useful during product reception.`
- Replace `By` with `When`